aspnet/conceptual-overview/dotnetcore.rst

```diff
@@ -79,7 +79,7 @@ Additional Reading
 Learn more about .NET Core:
 	- `Immo Landwerth Explains .NET Core <http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx>`_
 	- `What is .NET Core 5 and ASP.NET 5 <http://blogs.msdn.com/b/cesardelatorre/archive/2014/11/18/what-is-net-core-5-and-asp-net-5-within-net-2015-preview.aspx>`_
```

Comment:
After reading an article about all the renaming, I think the .Net Core 5 naming is intentionally left unchanged in this document. The intention is to not confuse people but I think the effect is exactly that. Especially for people new to ASP.NET this is very confusing. If the decision is made to go with .Net Core, as many articles as possible should be modified for the new naming.